packages/oml/oml.0.0.1/descr

```diff
@@ -0,0 +1,4 @@
+beginnings of an OCaml Math (and statiscs) Library
```

Comment:
I assume `statiscs` is supposed to be `statistics` so it is still missing a syllable.